packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -1020,6 +1029,8 @@ export function TransferPanelMain({
               return
             }
 
+            actions.app.setSelectedToken(null)
```

Comment:
we should also reset the amount to 0, otherwise users are left with the amount of token for ETH

Reply:
we discussed this offline and we need to fix query params rerendering before we can reset amount here